BUILD.bazel

```diff
@@ -0,0 +1,407 @@
+    defines = DEFINES,
+    includes = [half_path],
+    textual_hdrs = glob([
+        half_path + "half.h",
+        half_path + "halfExport.h",
+        half_path + "halfFunction.h",
+        half_path + "halfLimits.h",
+    ]) + [
+        ":toFloatAutoGen",
+        ":eLutAutoGen",
+    ],
+    deps = [":ilmbase_config"],
+)
+
+################################################################################
+# Build the Iex library
+iex_path = "IlmBase/Iex/"
+
+cc_library(
+    name = "iex",
+    srcs = glob([iex_path + "*.cpp"]),
+    hdrs = glob([iex_path + "*.h"]),
+    copts = all_options,
+    defines = DEFINES,
+    includes = [iex_path],
+    deps = [":ilmbase_config"],
+)
+
+################################################################################
+# Build the IexMath library
+iex_math_path = "IlmBase/IexMath/"
+
+cc_library(
+    name = "iex_math",
+    srcs = glob([iex_math_path + "*.cpp"]),
+    hdrs = glob([iex_math_path + "*.h"]),
+    copts = all_options,
+    includes = [iex_math_path],
+    deps = [
+        ":half",
+        ":iex",
+        ":ilmbase_config",
+    ],
+)
+
+################################################################################
+# Build the IlmThread library and its tests components
+ilm_thread_path = "IlmBase/IlmThread/"
+
+ilm_thread_win_srcs_pattern = ilm_thread_path + "*Win32.cpp"
+
+ilm_thread_win_hdrs_pattern = ilm_thread_path + "*Win32.h"
+
+cc_library(
+    name = "ilm_thread",
+    srcs = glob(
+        [ilm_thread_path + "*.cpp"],
+        exclude = [
+            ilm_thread_win_srcs_pattern,
+            ilm_thread_path + "*PosixCompat.cpp",
+            ilm_thread_path + "*Posix.cpp",
+            ilm_thread_path + "*OSX.cpp",
+        ],
+    ) + select({
+        ":windows_x86_64": glob([ilm_thread_win_srcs_pattern]),
+        ":linux_x86_64": [],
+    }),
+    hdrs = glob(
+        [ilm_thread_path + "*.h"],
+        exclude = [ilm_thread_win_hdrs_pattern],
+    ) + select({
+        ":windows_x86_64": glob([ilm_thread_win_hdrs_pattern]),
+        ":linux_x86_64": [],
+    }),
+    copts = all_options,
+    defines = DEFINES,
+    includes = [
+        ilm_thread_path,
+    ],
+    linkopts = select({
+        ":windows_x86_64": [],
+        ":linux_x86_64": ["-pthread"],
+    }),
+    deps = [
+        ":iex",
+        ":ilmbase_config",
+    ],
+)
+
+################################################################################
+# Build the Imath library
+imath_path = "IlmBase/Imath/"
+
+cc_library(
+    name = "imath",
+    srcs = glob([imath_path + "*.cpp"]),
+    hdrs = glob([imath_path + "*.h"]),
+    copts = all_options,
+    defines = DEFINES,
+    includes = [
+        imath_path,
+    ],
+    deps = [
+        ":half",
+        ":iex",
+        ":iex_math",
+        ":ilmbase_config",
+    ],
+)
+
+################################################################################
+# Build the IlmImf library
+ilm_imf_path = "OpenEXR/IlmImf/"
+
```

Comment:
This assumes the old code arrangement with various libraries in IlmBase, and the source for the OpenEXR library in IlmImf, instead of src/lib/OpenEXR. Would it be better to revise the build config so it works with the current master branch?

Reply:
Fixed